src/main/java/com/kakao/golajuma/vote/util/ImageUploader.java

```diff
@@ -60,4 +62,25 @@ public String uploadImageByBase64(CreateVoteRequest.OptionDto request) {
 			throw new ImageException("이미지 저장 실패");
 		}
 	}
+
+	public String getImage(String imagePath) {
+		if (imagePath == null || imagePath.equals("")) {
```

Comment:
```suggestion
		if (imagePath.isBlank()) {
```
isBlank는 null, "", " "까지 텅 빈 것으로 판단하고 isEmpty는 null, ""까지만 텅 빈 것으로 판단합니다.